core/server/api/v2/pages.js

```diff
@@ -43,7 +43,10 @@ module.exports = {
             'fields',
             'formats',
             'debug',
-            'absolute_urls'
+            'absolute_urls',
+            // NOTE: only for internal context
```

Comment:
These are required so that "frame" correctly processes internal options with transactions. If the context is not internal the options are automatically cleaned here - https://github.com/TryGhost/Ghost/blob/a31ed7c/core/server/api/shared/serializers/input/all.js#L36-L39